src/components/InputForm.js

```diff
@@ -38,13 +39,15 @@ const InputBooks = () => {
   };
 
   return (
-    <div>
-      <form onSubmit={createNewBook}>
-        <input type="text" name="Book" placeholder="Book" required value={title} onChange={BookInput} />
+    <footer className="footer">
+      <h1 id="add-book">ADD NEW BOOK</h1>
```

Comment:
- [ ] Kindly use the correct font size and color for this h1 element so that it matches the Zeplin design.

**This is how your design looks like:**

![image](https://user-images.githubusercontent.com/86318284/202715199-d2779dc2-cb34-447e-9970-5e4e542d021b.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/202715264-22cd8104-6327-422c-8abd-2991c87c31a9.png)